documentation/modules/proc-configuring-kafka-bridge-smr-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+[id='proc-configuring-kafka-bridge-smr-metrics-{context}']
+= Configuring Strimzi Metrics Reporter metrics
+
+[role="_abstract"]
+Enable metrics with Strimzi Metrics Reporter by setting the `bridge.metrics` option to `strimziMetricsReporter`.
+
+.Prerequisites
+
+* xref:proc-downloading-kafka-bridge-{context}[The Kafka Bridge installation archive is downloaded].
+
+.Procedure
+
+. Set the `bridge.metrics` configuration to `strimziMetricsReporter`.
++
+.Configuration for enabling metrics
+
+[source,properties]
+----
+bridge.metrics=strimziMetricsReporter
+----
++
+Optionally, you can set a comma-separated list of regexes used to filter exposed metrics using the `kafka.prometheus.metrics.reporter.allowlist` property.
+If not set, all available metrics will be exposed.
```

Comment:
Didn't we discussed in the past that we will have some more reasonable default than `.*`?

Reply:
That was discussed as part of the operator changes, but I think we can also apply that logic here.

Reply:
So, we now have some different defaults in the code so this should be updated as well as I guess it is not all available metrics anymore?